### ISSUE TITLE
Unable to move the hands with keyboard while using NVDA 

### ISSUE THREAD

**Issue by Matthew-Moore240** [phetsims/ratio-and-proportion#46]
Neither one of the hands work with keyboard while using NVDA. Instead when they are on focus they scroll through the PDOM with the arrow keys.

**Comment by jessegreenberg** [comment]
I think that #44 should be done before we debug this. The representation in the PDOM is going to change completely and may fix this on its own. Removing my assignment, but feel free to reassign if I can help!

**Comment by zepumph** [comment]
This was fixed by #47. @Matthew-Moore240 please confirm and close.
